### ISSUE TITLE
Segmentation Issue with Multiple Selected Segments

### ISSUE THREAD

**Issue by FrnklyN** [kijai/ComfyUI-segment-anything-2#13]
I am encountering an issue where, when multiple segments are selected, only the first segment is processed and outputted.
How can I ensure that all selected segments are segmented and processed at once?

**Comment by kijai** [comment]
Yeah every point so far has been added to same object, I didn't work on the multi object yet.

I added very simple way to to use that now, a switch that makes it treat every point as separate object:

https://github.com/user-attachments/assets/b5647167-34d6-497d-b703-516f35f1265c

**Comment by Erwin11** [comment]
What is the name of the canvas node in your video?

**Comment by kijai** [comment]
> What is the name of the canvas node in your video?

It's not really ready for proper use yet, just something I've been working on. It's in KJNodes for testing already though, called PointsEditor.
